part_1/1.4_conditionals.md

```diff
@@ -57,10 +67,222 @@ When you run your code, if the age condition is met you should see the sentence
+
+An else statement used to define a certain type of flow. It follows your typical if -> then scenarios. If the condition of the `If` statement is not met then run the code contained in the `Else` statement.
+
+Let's try using an `Else` statement in the movie theatre problem.
+
+**Problem:**
+
+The local movie theatre now wants to recommend *The Princess Bride* for all viewers who are under 17. 
+
+Let's start with our old code. 
+
+```go
+package main
+
+import "fmt"
+
+func main() {
+		age := 21
+		if age >= 17 {
+				fmt.Println("You are allowed to buy a ticket)
+		}
+}
+```
+We just need to add an `Else` statement to recommend *the Princess Bride.*
+
+```go
+		if age >= 17 {
+				fmt.Println("You are allowed to buy a ticket")
+		} else {
+			fmt.Println("Why don't you try the Princess Bride instead")
+		}
+```
+What happens when you change the age from 21 to 16? Does it print the right line of code? 
+
+*PRO TIP: Expected Behavior is when code does what you expect it to do. The outcome of a simple function should always have an 
+expected behavior. If it does something you don't expect then it is most likely in need of a rewrite. (In [part_5](part_5/5.2_unit_tests.md) we will
+explore using tests to define Expected Behavior.*
+
+## Else If Statements
+
+Sometimes when programming we are given problems where we have to define multiple patterns of behavior based on met conditions. We going to 
+continue using the *movie theater* problem to explore this. 
+
+**Problem:**
+
+The local movie theatre has noticed that some small children do not enjoy watching *The Princess Bride*. The movie theatre wants to offer tickets
+to *The Little Mermaid* for patrons who are 7 years old and younger. 
+
+How do we do this?
+
+Using only the tools we have so far the code might look something like this. 
+
+```go 
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	age := 10
+	if age >= 17 {
+		fmt.Println("You are allowed to buy a ticket")
+	} else {
+		if age >= 7 {
+			fmt.Println("Would you like to watch the Little Mermaid")
+		} else {
+			fmt.Println("Why don't you try the Princess Bride instead")
+		}
+	}
+
+}
+```
+
+This works, but doesn't this look kinda gross? This code is hard to read and hard to know what the expected behavior is supposed to be. When code is hard to read the first thing we should ask ourselves is if there is better way to write this? 
+
+In Go, and most programming language, there is a construct called [`Else If`](https://gobyexample.com/if-else) that allows us to write this is a more readable way. `Else If` allows us to add additional condition checks to an `If Els` statement. For example: 
+
+```go
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	age := 6
+	if age >= 17 {
+		fmt.Println("You are allowed to buy a ticket")
+	} else if age <= 7 {
```

Comment:
Are the change of `age` and `age <= 7` compared to the previous example deliberate? I think it makes the larger point about `else if` more confusing.

Reply:
no... i may have copy pasted testing code... I will adjust... I think example consistancy is important in this case